src/main/java/drug_request_use_case/DrugRequestDsInvokeModel.java

```diff
@@ -0,0 +1,36 @@
+package drug_request_use_case;
+
+import java.time.LocalDateTime;
+
+public class DrugRequestDsInvokeModel {
+    //consider if it's necessary to use Final or not, is drugName expected to be modified
```

Comment:
I think you're correct. however the reasoning behind why we should keep it final is a bit different i think.
I was thinking rather than the user manually type in drug name we would use a combo Box with a drop down list from which all available drugs can be selected. In this case user does not type in drug name and the drug sleected will always be spelled correctly. 

However this leads to another use case when the drug they want is not available and they need to request one then they would type the drug name into another field. This is an entirely separate use case which I have not implemented yet but would be very similar to what i have so far.